Yatima/Datatypes/Expr.lean

```diff
@@ -1,204 +1,25 @@
 import Yatima.Datatypes.Univ
+import YatimaStdLib.Ord
+import Lurk.Field
 
-namespace Yatima
+namespace Yatima.IR
 
-namespace IR
+instance (priority := high) : Hashable Literal where hash
```

Comment:
What is the reason behind the `priority := high`?

Reply:
Just to make sure that we won't use the instance that's defined in core

Reply:
Is the instance defined in core slow? I think it would be faster

Reply:
It's not much about the speed. Wee need this instance to avoid the hash collision we were getting earlier

Reply:
Ah, I see, very tricky indeed